rele/publishing.py

```diff
@@ -4,6 +4,28 @@
 
 
 def publish(topic, data, **kwargs):
+    """Shortcut method to publishing data to PubSub.
+
+    This is a shortcut method that instantiates the Publisher if not already
```

Comment:
Alternative: "Publishes a message using a global Publisher to ensure long running processes don't leak memory"